src/api/connection.js

```diff
@@ -62,26 +64,26 @@ Connection_.prototype.handleResponse = function (responseCode, responseBody, que
 }
 
 Connection_.prototype.graphQLQuery = function (query, queryName) {
+  const response = this.fetchQuery(query)
   try {
-    var response = this.fetchQuery(query)
-    var responseCode = response.getResponseCode()
-    var responseBody = JSON.parse(response.getContentText())
+    const responseCode = response.getResponseCode()
+    const responseBody = JSON.parse(response.getContentText())
 
-    var logMessage = {
+    const logMessage = {
       type: 'RequestLog',
-      query: query,
-      queryName: queryName
+      query,
+      queryName
     }
 
     logInfo_(logMessage)
 
     return this.handleResponse(responseCode, responseBody, queryName)
   } catch (e) {
-    var error = {
+    const error = {
       type: e.name,
       message: e.message,
-      query: query,
-      queryName: queryName,
```

Comment:
Leve also these 2, it's weird to have 2 key-value pairs followed by 2 variables and then again followed by 2 key-value pairs